.github/workflows/buildcheck.yml

```diff
@@ -20,21 +21,31 @@ jobs:
     needs: call-reusable-workflow
     runs-on: ubuntu-latest
     steps:
-    - name: Restore cached tags and jars
-      id: cache-tags-jars
-      uses: actions/cache@v4
+    - name: Set up JDK for x64
```

Comment:
In jedem Job muss das JDK explizit konfguriert werden. Diese Settings vererben sich von Job zu Job.

Reply:
Das ist gut. So könnte man nightlies mit einer höheren Java-Version bauen, als die bisherigen releases.